scripts/run_ui_tests.sh

```diff
@@ -0,0 +1,48 @@
+sudo apt update
+apt install net-tools
+apt-get install wget
```

Comment:
We're performing Ubuntu-specific operations, we should either move this to the `.github` directory, or make this script suitable to work with Unixes.